src/components/LoginSignup/SignIn.js

```diff
@@ -63,8 +83,67 @@ function SignIn() {
     }
   };
 
+  const onResendLinkClicked = async () => {
+    resendLink(emailValue, passwordValue);
+    setOpenReset(false);
+    setOpenSuccess(true);
+  };
+
+  const action = (
+    <>
+      <Box m={0.5}>
+        <Button color="inherit" size="small" disableRipple onClick={() => setOpenReset(false)}>
+          Abbrechen
+        </Button>
+      </Box>
+      <Box m={0.5}>
+        <Button color="secondary" size="small" disableRipple onClick={onResendLinkClicked}>
+          Bestätigen
+        </Button>
+      </Box>
+    </>
+  );
+
   return (
     <>
+      {location.state.signUpSuccessful && (
```

Comment:
```suggestion
      {location?.state?.signUpSuccessful && (
```
Beim Direkteinsprung auf "/signin" crashed die App, weil vermutlich `signUpSuccessful` nicht mit kommt. Kommt ja auch nur nach der Registrierung mit. 

Die Suggestion sollte das Problem lösen. Müsste man aber nochmal testen :)

Reply:
PS: Haben wir das Problem auch bei der Trail App? :D